packages/react-components/src/components/UnsupportedEnvironment.tsx

```diff
@@ -37,11 +42,14 @@ export interface UnsupportedEnvironmentProps {
   onTroubleshootingClick?: () => void;
   /** String overrides for the component */
   strings: UnsupportedEnvironmentStrings;
+  /** Allow user to continue into the call */
```

Comment:
> Allow user to continue into the call
Isn't quite right, this is a dumb component that doesn't actually do anything, just allows Contoso to do something, consider:
```suggestion
  /** Callback for the "Continue Anyway" button. Use this to provide a mechanism to allow the user to continue into the call */
```